cfgov/unprocessed/css/atoms/partially-styled-link.less

```diff
@@ -0,0 +1,50 @@
+/* topdoc
+  name: Partially styled link
+  family: cfgov-atoms
+  patterns:
+    - name: Default example
+      markup: |
+          <a class="a-partially-styled-link">
+            <span class="a-partially-styled-link_unstyled">
+              Unstyled text
+            </span>
+            <span class="a-partially-styled-link_styled">
+              Underlined text
+            </span>
+          </a>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          a.a-partially-styled-link
+            span.a-partially-styled-link_unstyled
+            span.a-partially-styled-link_styled
+  tags:
+    - cfgov-atoms
+*/
+
+.a-partially-styled-link {
```

Comment:
To follow the BEM Standard, this should be something like `a-link a-link__partially-styled`